### DISCUSSION TITLE
Made a py script for scaning and removing IP or MAC address

### DISCUSSION THREAD

**Discussion by laxduke** [netalertx/NetAlertX#916] [Ideas]
Hi.
Just made a script for checking and deleting ip / mac address entries from database.
Got some garbage and put my Cursor with Claude to good use
Please check the code and let's add the feature to the main programm.

All the best

[netalertx_cleanup.zip](https://github.com/user-attachments/files/18216072/netalertx_cleanup.zip)

**Comment by jokob-sk** [comment]
Hey @laxduke ,

Thanks for that, I created a scripts folder - let's see if I get more contributions. Keeping it here for now: https://github.com/jokob-sk/NetAlertX/tree/main/scripts/db_cleanup
